packages/client/lib/functions/ProofUtil.ts

```diff
@@ -1,4 +1,13 @@
-import { BAD_PARAMS, JWS_NOT_VALID, Jwt, JWTPayload, ProofOfPossession, ProofOfPossessionCallbacks, ProofType } from '@sphereon/openid4vci-common';
+import {
+  BAD_PARAMS,
+  JWS_NOT_VALID,
+  Jwt,
+  JWTPayload,
+  ProofOfPossession,
+  ProofOfPossessionCallbacks,
+  ProofType,
+  Typ,
+} from '@sphereon/openid4vci-common';
 import Debug from 'debug';
 import { JWTHeaderParameters } from 'jose';
```

Comment:
Please remove the  jose import here copy JwtHeader params from jose or from the openid4vci-common as that also already seems to contain Jwt and JWTPayload anyway, it probably also has the concept of Jwt Headers

I want to stay away from using jose in non-dev imports, as certain jose functions do not work in RN environments.

I know this is not part of this PR, but I notice it now and this has the potential to lead to problems down the road.

Reply:
Fixed!